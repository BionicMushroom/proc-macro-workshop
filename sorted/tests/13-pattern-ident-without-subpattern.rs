#[sorted::check]
fn main() {
    let x = 10;

    #[sorted]
    match x {
        0 => println!("0"),
        ref num => println!("{num}"),
    }
}
