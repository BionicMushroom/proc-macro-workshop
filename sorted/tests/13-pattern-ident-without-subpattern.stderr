error: `ref [mut]` patterns that bind variables without `@ <subpattern>` are unsupported by #[sorted]
 --> tests/13-pattern-ident-without-subpattern.rs:8:9
  |
8 |         ref num => println!("{num}"),
  |         ^^^^^^^
